src/SUMMARY.md

```diff
@@ -4,7 +4,10 @@
 
 - [The Geo Engine](./geoengine/intro.md)
 
+  - [Collections](./geoengine/collections.md)
```

Comment:
I would explain stuff in this order:
- Workflows,
- Datasets,
- Layers and Layer Collections

I wouldn't use "Collections" as a term on its own, because there may be other collections in the system (in the future)

Reply:
1. For now, I would keep alphabetical ordering to look up stuff easier. We can discuss this for later when we have more stuff in there and maybe some story behind it.
2. [96f675f](https://github.com/geo-engine/docs/pull/44/commits/96f675f2c13a7be11b8e5ca8825ea0c60c79f1b9)